National Gallery of Art - USA.js

```diff
@@ -1,8 +1,8 @@
 {
 	"translatorID": "ed28758b-9c39-4e1c-af89-ce1c9202b70f",
-	"label": "National Gallery of Art - U.S.A.",
+	"label": "National Gallery of Art - USA",
```

Comment:
Change the label here as well to be consistent with the file name (otherwise Scaffold will always trying to rename it).